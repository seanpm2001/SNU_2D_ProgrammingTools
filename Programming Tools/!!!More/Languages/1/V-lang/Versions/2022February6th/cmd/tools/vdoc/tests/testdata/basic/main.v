pub const (
	source_root = 'temp'
)

// funky - comment for function below
pub fn funky() {
	println('hi')
}
