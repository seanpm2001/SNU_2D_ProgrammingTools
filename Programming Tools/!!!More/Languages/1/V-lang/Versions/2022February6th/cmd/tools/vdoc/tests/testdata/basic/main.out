module main

const (
	source_root = 'temp'
)
fn funky()